tdrs-backend/.env.example

```diff
@@ -0,0 +1,49 @@
+### Local Development Environment
+# Copy this file to `.env` and replace variables as needed
+#
+
+DISABLE_COLLECTSTATIC=1
+
+# Private JWT Key used to generate the client assertion
+JWT_KEY=a_secret_key
+
+#Base64 encoded public cert used for CI/CD pipeline testing
+JWT_CERT_TEST=a_public_cert
+
+# Add secret key from "Development Credentials" document
+SECRET_KEY=secret
+DJANGO_SECRET_KEY=secret
+
+# Parameters defined in the login.gov documentation https://developers.login.gov/oidc/
+CLIENT_ID=urn:gov:gsa:openidconnect.profiles:sp:sso:hhs:tanf-proto-dev
+ACR_VALUES=http://idmanagement.gov/ns/assurance/ial/1
+OIDC_OP_ISSUER=https://idp.int.identitysandbox.gov/
+OIDC_OP_JWKS_ENDPOINT=https://idp.int.identitysandbox.gov/api/openid_connect/certs
+OIDC_OP_AUTHORIZATION_ENDPOINT=https://idp.int.identitysandbox.gov/openid_connect/authorize
+OIDC_OP_LOGOUT_ENDPOINT=https://idp.int.identitysandbox.gov/openid_connect/logout
+OIDC_OP_USER_ENDPOINT=https://idp.int.identitysandbox.gov/api/openid_connect/userinfo
+CLIENT_ASSERTION_TYPE=urn:ietf:params:oauth:client-assertion-type:jwt-bearer
+
+# Override this endpoint as defined in "Development Credentials" document
```

Comment:
The "Development Credentials" document is referenced a few times -- I don't see a "Development Credentials" document included in this PR?

Reply:
To avoid committing any secrets or environment specific values to source control I split these off into a separate MD file, which can be shared as needed outside of git through me or @carltonsmith. I'm definitely open to suggestions on the best way to handle values for secrets, keys and such; but to keep with `dotenv` best practices we shouldn't commit the explicit values. Storing these in something like Vault or AWS KMS long term would be a good way to share to specific users but may be overkill depending on how many people will be working on this project.

Would it make more sense to instead document how to generate these values and configure them on the other side (ie. Login.gov, etc) and not mention the credentials document at all?

Reply:
@jtwillis92 Very much agreed that keeping secrets outside of source control is fundamental! 

I just didn't know what document you were referring to in this code comment.

I'd suggest something like "Ask the project Tech Lead or Product Manager for the development credentials document" so that new folks know how to get/find this document.